core/src/main/java/io/questdb/griffin/engine/functions/bool/InLongFunctionFactory.java

```diff
@@ -115,9 +117,14 @@ public Function newInstance(
                                     2)
                     );
                 default:
-                    LongHashSet inVals = new LongHashSet((int) ((argCount / LongHashSet.DEFAULT_LOAD_FACTOR)));
-                    parseToLong(args, argPositions, inVals);
-                    return new InLongConstFunction(args.getQuick(0), inVals);
+                    DirectLongHashSet inVals = new DirectLongHashSet(argCount, DirectLongHashSet.DEFAULT_LOAD_FACTOR, MemoryTag.NATIVE_FUNC_RSS);
```

Comment:
nit: instead of providing `DirectLongHashSet.DEFAULT_LOAD_FACTOR` value we could add one more constructor to the `DirectLongHashSet` class: `DirectLongHashSet inVals = new DirectLongHashSet(argCount, MemoryTag.NATIVE_FUNC_RSS);`

Reply:
fixed in https://github.com/questdb/questdb/pull/6439/commits/a85522e991140ba780d8126df211eff33bae6a37